src/ripple/app/tx/impl/MPTokenAuthorize.cpp

```diff
@@ -116,107 +133,106 @@ TER
 MPTokenAuthorize::doApply()
 {
     auto const mptIssuanceID = ctx_.tx[sfMPTokenIssuanceID];
-    auto const sleMptIssuance = view().read(keylet::mptIssuance(mptIssuanceID));
-    if (!sleMptIssuance)
-        return tecINTERNAL;
-
     auto const sleAcct = view().peek(keylet::account(account_));
     if (!sleAcct)
         return tecINTERNAL;
 
     auto const holderID = ctx_.tx[~sfMPTokenHolder];
     auto const txFlags = ctx_.tx.getFlags();
```

Comment:
I think you can just use `ctx_.tx.getFlags()` inline because the flags is used only once within each block.